lib/kv-redis.js

```diff
@@ -368,3 +381,18 @@ function(modelName, filter, options, callback) {
     });
   }
 };
+
+/**
+ * Get model name for a given model
+ * @param {String} modelName The model name
+ * @returns {String} model name
+ */
+RedisKeyValueConnector.prototype.modelName = function(modelName) {
```

Comment:
Nitpick: function & method names usually start with a verb, e.g. `getModelName()`. I know we are not consistently following this rule in our codebase, therefore I am ok to keep `modelName()` if you prefer.